py/selenium/webdriver/remote/remote_connection.py

```diff
@@ -387,7 +404,7 @@ def _request(self, method, url, body=None):
             try:
                 self._conn.request(method, parsed_url.path, body, headers)
                 resp = self._conn.getresponse()
-            except httplib.HTTPException:
+            except (httplib.HTTPException, socket.error):
```

Comment:
I hesitated between ```socket.error``` or ```socket.timeout```. And I prefer ```socket.error```, because sometimes chromedriver raises ```peer to peer error``` and it's not a ```socket.timeout```